engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/gameDetailsScreen/GameDetailsScreen.java

```diff
@@ -118,14 +119,17 @@ public void initialise() {
         worldDescription = find("worldDescription", ScrollableArea.class);
         descriptionContainer = find("descriptionContainer", ScrollableArea.class);
         openModuleDetails = find("openModuleDetails", UIButton.class);
+        previewSlideshow = find("preview", UIImageSlideshow.class);
 
         if (descriptionContainer != null && worldDescription != null && descriptionTitle != null &&
-                gameModules != null && gameWorlds != null && biomes != null && blocks != null && openModuleDetails != null) {
+                gameModules != null && gameWorlds != null && biomes != null && blocks != null && openModuleDetails != null ||
```

Comment:
Should this not be `&& previewSlideshow != null`, not `||`

Reply:
thanks, fixed